src/frontmatter.xml

```diff
@@ -35,84 +35,87 @@
+
+        </paragraphs>
+
+        <paragraphs>
+            <title>Accessibility</title>
+            <p>The HTML version is intended to meet or exceed all web accessibility standards. If you encounter an accessibility issue, please report it to the editor.</p>
+            <ul>
+                <li><p>All graphs and images should have meaningful alt text that communicates what a sighted person would see, without necessarily giving away anything that is intended to be deduced from the image.</p></li>
+                <li><p>All math content is rendered using MathJax. MathJax has a contextual menu that can be accessed in several ways, depending on what operating system and browser you are using. The most common way is to right-click or control-click on some piece of math content.</p></li>
+                <li><p>In the MathJax contextual menu, you may set options for triggering a zoom effect on math content, and also by what factor the zoom will be.</p></li>
+                <li><p>If you change the MathJax renderer to MathML, then a screen reader will generally have success verbalizing the math content.</p></li>
+            </ul>
+        </paragraphs>
+
+        <paragraphs>
+            <title>Tablets and Smartphones</title>
+            <p>MathBook XML documents like this text are <q>mobile-friendly</q>. The display adapts to whatever screen size or window size you are using. A math teacher will always recommend that you do not study from the small screen on a phone, but if it's necessary, this text gives you that option.</p>
+        </paragraphs>
+    </preface>
+
+    <preface xml:id="to-the-student">
+        <title>To the Student</title>
+
+        <p>You may think of this text as a <q>workbook</q>. Each chapter begins with activity sets, meant to serve as an introduction to a topic, and your instructor may assign activities to be done out of class as homework. In this way, you will come to class, having already studied the basics of a new topic, and ready to build on that new knowledge.</p>
+        <p>The activity sets are followed by a <q>gist</q> section. The gist assumes that you have completed the activities, and it provides a retelling of the basic concepts and techniques. This will be a good place for you to refer when studying the material for an exam or quiz.</p>
+        <p>Instead of just <em>reading</em> example problems, you will get to explore new ideas by working along <em>with</em> the examples. Most problems in the book allow you to manipulate dynamic graphs and receive immediate feedback by submitting your answers through WeBWorK online.</p>
```

Comment:
Wherever "WeBWorK" comes up, I recommend using `<webwork />` which will get the capitalization right without inevitable human errors. (This is what a webwork tag does when there is nothing inside it.)